### ISSUE TITLE
after train_yolov3_lite.sh，way can't test the object

### ISSUE THREAD

**Issue by zhuky888888** [eric612/MobileNet-YOLO#14]
hi @eric612,
    I'm the new hand， i modify the file mobilenet_yolov3_lite_solver.prototxt in models/yolov3 , like this:
    test_iter: 6  test_interval:10 base_lr:0.0005 max_iter:25 snapshot:10 
   after train about 10 iters, i optimize the mobilenet_yolov3_lite_deploy_iter_10.caffemodel to optimize the mobilenet_yolov3_lite_deploy.caffemodel maded by merge_bn.py.
   then i use the command "sh demo_yolov3_lite.sh", but i can't the object anyone . why ?
    and in the stage of train, i find the avg_noobj/avg_obj/avg_iou/avg_cat/recall and detect_eval are so small, but the loss  is bigger about 1336.13. why ?  it do not work with the pretrained model "models/MobileNet/mobilenet_iter_73000.caffemodel"

**Comment by eric612** [comment]
Here is my python code , you can try this first 
https://drive.google.com/file/d/1Ul7r8swy_nNoPrVYvMIrK-1Z4ROsQiSF/view?usp=sharing

The caffemodel and prototxt were put in 
https://github.com/eric612/MobileNet-YOLO/tree/master/models/yolov3

**Comment by zhuky888888** [comment]
I find the answer, models/MobileNet/mobilenet_iter_73000.caffemodel in the train_yolov3_lite.sh is not used for yolov3_lite, when i substite the models/MobileNet/mobilenet_iter_73000.caffemodel with models/yolov3/mobilenet_yolov3_lite_deploy_iter_68000.caffemodel, it workes ok. thanks.

**Comment by zhuky888888** [comment]
hi eric，
   还想请教个预训练的问题，我这边只需要检测1类对象，而mobilenet-yolo提供的是针对20类对象的模型参数，除了将num_output改成(1+4+2)*3 = 21外，我该如何修改prototxt比较合适？

**Comment by zhuky888888** [comment]
hi @eric612，
还想请教个预训练的问题，我这边只需要检测1类对象，而mobilenet-yolo提供的是针对20类对象的模型参数，除了将num_output改成(1+4+2)*3 = 21外，我该如何修改prototxt比较合适？

**Comment by eric612** [comment]
應該沒問題，除非效能不好，再來考慮修改其他部分，但一般來說只有一個class的話，效率上有些浪費，不需要做class的regression。
